test/e2e/certificates_test.go

```diff
@@ -267,6 +269,69 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate in GCP mint/passthrough clusters-65035", func() {
+
+			By("Getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
```

Comment:
Just to clarify, we're skipping this test if Infrastructure spec doesn't match platform type GCP, right?
Need suggestions, if this is a good approach to do :) plausibly other test examples using a similar `Skip()` would help support IMO. 

Alternatively, please check if we could use some sort of labels within ginkgo to filter out basis of platform and in our `make test-e2e` we can pass the filter arg in some way to ensure we're intending to run on AWS/GCP. IMHO, that would serve as better long-term solution as the number of test cases for specific cloud platform could grow over time and hence, it be easy to filter from the entrypoint of the tests than within the tests itself.

Reply:
@swghosh Yes, this 'Skip' just using for filter test case doesn't match platform type GCP, 
we also think about use labels within ginkgo to filter the platfrom, but there is not good way to realize it.